nvflare/app_opt/tracking/wandb/wandb_receiver.py

```diff
@@ -35,9 +34,9 @@ class WandBTask(NamedTuple):
 
 
 class WandBReceiver(AnalyticsReceiver):
-    def __init__(self, kwargs: dict, mode: str = "offline", events=None, process_timeout=10):
-        if events is None:
-            events = ["fed.analytix_log_stats"]
```

Comment:
are we intentionally removing setting events to ["fed.analytix_log_stats"] by default?